core/domain/story_domain.py

```diff
@@ -0,0 +1,359 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects relating to stories."""
+
+from constants import constants
+from core.platform import models
+import feconf
+
+(story_models,) = models.Registry.import_models([models.NAMES.story])
+
+
+# Do not modify the values of these constants. This is to preserve backwards
+# compatibility with previous change dicts.
+STORY_PROPERTY_TITLE = 'title'
+STORY_PROPERTY_DESCRIPTION = 'description'
+STORY_PROPERTY_TOPIC = 'topic'
```

Comment:
Hm, this one is interesting. I consider it more of a change to a topic; this property is just a backlink to be kept updated with such a change. Which makes the commit process more interesting, since what happens when you revert the change to the topic but not the story, or vice versa?

We might need to reconsider the design of how we store the topic-to-story mapping. That particular part of a topic doesn't really need to have versioning. Maybe topics aren't versioned and we maintain the info there. Or maybe we forbid reverts for topics...

Not sure. Let's discuss when we meet.

Reply:
For now, I removed this field.